Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM nginx:1.13
 EXPOSE 8080
-ADD index.html /usr/share/nginx/html/
+ADD _build /usr/share/nginx/html/
```

Comment:
I would suggest using COPY instead of ADD (to avoid issues in the future due to ADD magic)